File: tmp/nx-e2e/proj/apps/subdir/boot-maven-app-4331202/src/main/java/com/jnxplus/bootmavenapp4331202/ServletInitializer.java
package com.jnxplus.bootmavenapp4331202;

import org.springframework.boot.builder.SpringApplicationBuilder;
import org.springframework.boot.web.servlet.support.SpringBootServletInitializer;

public class ServletInitializer extends SpringBootServletInitializer {

  @Override
  protected SpringApplicationBuilder configure(
    SpringApplicationBuilder application
  ) {
    return application.sources(SubdirBootMavenApp4331202Application.class);
  }
}

File: tmp/nx-e2e/proj/apps/boot-maven-app-4532651/src/main/kotlin/com/example/bootmavenapp4532651/ServletInitializer.kt
package com.example.bootmavenapp4532651

import org.springframework.boot.builder.SpringApplicationBuilder
import org.springframework.boot.web.servlet.support.SpringBootServletInitializer

class ServletInitializer : SpringBootServletInitializer() {

  override fun configure(application: SpringApplicationBuilder): SpringApplicationBuilder {
    return application.sources(BootMavenApp4532651Application::class.java)
  }
}
